weblate/utils/search.py

```diff
@@ -69,16 +69,16 @@ class Exact(whoosh.query.Term):
     pass
 
 
-class ExactPlugin(QuotePlugin):
-    """Exact match plugin with quotes to specify an exact term."""
+class ExactPlugin(whoosh.qparser.TaggingPlugin):
+    """Exact match plugin to specify an exact term."""
 
     class ExactNode(whoosh.qparser.syntax.TextNode):
         qclass = Exact
 
         def r(self):
             return "Exact %r" % self.text
 
-    expr = r"\=(^|(?<=\W))['\"](?P<text>.*?)['\"](?=\s|\]|[)}]|$)"
+    expr = r"\=(^|(?<=\W))(['\"]?)(?P<text>.*?)\2(?=\s|\]|[)}]|$)"
```

Comment:
This seems to break the tests: `AssertionError: <Q: (AND: ('source__iexact', 'hello'))> != <Q: (AND: ('source__substring', '=hello'))>`, see https://github.com/WeblateOrg/weblate/pull/3677/checks?check_run_id=563709490#step:8:575

Reply:
Yeah. I guess it should be equal. :thinking:

Reply:
Previously, quotes were needed, so I added that test.